YDotNet/YDotNet.csproj

```diff
@@ -16,6 +16,18 @@
     <InternalsVisibleTo Include="$(AssemblyName).Tests.Unit" />
   </ItemGroup>
 
+  <ItemGroup>
+    <None Update="yrs.dll">
```

Comment:
I don't understand this change. The binaries are relatively large (e.g. when used in a mobile app or so). Therefore I created the native packages. If you want to bundle everything, the native packages are obsolete.